plugins/hosts/linux/cap/os_arch.rb

```diff
@@ -0,0 +1,20 @@
+include Vagrant::Util
+
+module VagrantPlugins
```

Comment:
This file is responsible for giving hardware architecture like `x86_64` or `i386`. Best practice for such code is in capabilities.